src/Combinatorics/Matroids/ChowRings.jl

```diff
@@ -0,0 +1,222 @@
+export chow_ring, augmented_chow_ring, select
+
+@doc Markdown.doc"""
+    chow_ring(M::Matroid, ring::MPolyRing=nothing; extended=false)
+
+Return the Chow ring of a matroid, optional also with the simplicial generators.
```

Comment:
```suggestion
Return the Chow ring of a matroid, optionally also with the simplicial generators.
```
But how does one trigger this optional Behavior? Perhaps with the options `ring` and `extended`? But how?

Conversely: either describe the purpose of `ring` and `extended`, or, if they are supposed to be "for internal use only", omit them here.